renovate.json

```diff
@@ -4,15 +4,17 @@
   ],
   "baseBranches": ["main"],
   "useBaseBranchConfig": "merge",
+  "commitMessageExtra": "",
+  "prBodyColumns": ["Package"],
```

Comment:
These are needed to hide the incorrect information renovate knows about the generated bump. Renovate does the bump to the latest version regardless the post update tasks and try to record that into the commit message. That bump is overwritten by the make force-dep post update task.